### PR TITLE
Moves case studies from Private Key section to a dedicated section

### PR DESCRIPTION
**Pull request by GBKS**
Also splits the four case studies into dedicated pages.

No changes were made to the case study content. Only addition was a landing page that lists the case studies and has a short description for each one.

I find this helpful to get a feel for what we already have in terms of case studies, what seems missing, etc.

### CODE REVIEW

**Comment by pavlenex** [comment]
Looks a bit weird without the nesting imo. The terminology isn't really self-explanatory and as `case-study` grows it'll be hard to distinguish perhaps what's connected to what.
For example, one might expect that the Multisig wallet is a case study about multisig wallets, but it's more focused on private key management aspect of it.

Screenshot from 2021-03-04 11-25-13

**Comment by GBKS** [comment]
@pavlenex there is nesting. Is is too subtle to see?

Agree about the naming, that needs to be clarified without becoming super long. We have to keep in mind though that this is navigation, and that people can go to the "Case study" landing page for longer descriptions. So this short nav item doesn't have to cover everything (should be more precise than it currently is though).

**Comment by pavlenex** [comment]
@GBKS I mean, as we add stuff, it needs to be categorized. 
- Case Studies
  - Private Key Management 
        -  Cloud
        -  Multisig
        -  Upgradable
        -  Shared 
   - Another category

**Comment by GBKS** [comment]
@pavlenex got it, let's tackle that separately. I'll add two-level nesting as a new issue as we don't have that functionality yet. I'm also wondering if we won't have overlap between case studies - there might be one that deals with both onboarding, private keys and accessibility, for example. Not sure how we'd organize that.

**Comment by danielnordh** [comment]
Looks good to me. Minor comments:

- Titles of the case studies (intro page, nav bar) should probably match up with the subpages
- PR #144 has a link to the existing case studies page in the 'Shared schemes' page that will need to be changed if this merges first (easy to do)

**Review by danielnordh — APPROVED** [review]
Looking good, approved.

File: guide/case-studies/cloud-backup.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Cash account / Daily spending
+
+Imagine a product which tries to solve the problem of quickly and easily sending smaller amounts of money to friends and family, or for small purchases. Ease and speed of use will be important as usage is likely to be on mobile devices and on the go. Users are not expected to be well versed in bitcoin technology or advanced private key management, which makes it reasonable to worry more about self-inflicted loss than from theft.
+
+A single-key scheme with [automatic cloud backup]({{ '/guide/private-key-management/single-user-schemes/#automatic-cloud-backup' | relative_url }}) might be the best choice for the majority of users in this case. For more advanced users you could offer the option to opt-out of automatic cloud backup and instead use a recovery phrase.
+
+{% include prototype.html
+   link = "https://www.figma.com/proto/SRWlaxbDulsacpPQn2TTri/Case-study-prototypes?node-id=1%3A3&viewport=1357%2C576%2C1&scaling=scale-down"
+   image = "/assets/images/guide/case-studies/case-cash.png"
```
Comment (ConorOkus) [inline guide/case-studies/cloud-backup.md:25]: Might be for another PR and I think it's been mentioned before, but should we start using SVG's instead so we can simplify and use one image now?

**Review by ConorOkus — APPROVED** [review]

**Review by pavlenex — APPROVED** [review]

File: guide/case-studies/cloud-backup.md (hunk 1 above)
Comment (GBKS) [reply guide/case-studies/cloud-backup.md:25]: Good call. We should do a review of this across the whole guide before we go live with V1.
